include/sound/sof/header.h

```diff
@@ -53,6 +53,7 @@
 #define SOF_IPC_GLB_TEST_MSG			SOF_GLB_TYPE(0xBU)
 #define SOF_IPC_GLB_PROBE			SOF_GLB_TYPE(0xCU)
 #define SOF_IPC_GLB_DEBUG			SOF_GLB_TYPE(0xDU)
+#define SOF_IPC_GLB_INPUT_EVENT			SOF_GLB_TYPE(0xFU)
```

Comment:
wouldn;t this be a major ABI change

Reply:
Yes, it is. This is just a proof of concept. I think the final version should be implemented for IPC4 in the first place, but I do not understand that well enough yet (working on it ATM).